app/views/kaminari/_paginator.html.erb

```diff
@@ -6,11 +6,25 @@
     remote:        data-remote
     paginator:     the paginator that renders the pagination tags inside
 -%>
-<%= paginator.render do -%>
-  <nav id="show-more-documents" role="navigation">
-    <ul class="previous-next-navigation">
-      <%= prev_page_tag unless current_page.first? %>
-      <%= next_page_tag unless current_page.last? %>
-    </ul>
-  </nav>
-<% end -%>
+
+<%
+  nav = {}
+
+  unless current_page.last?
+    nav[:next_page] = {
+      url: paginator.next_page_tag.url,
+      title: raw("Next <span class='visuallyhidden'>page</span>"),
```

Comment:
🤔 Based on gem use I'm going to suggest
`Next page` rather than  `Next <span class='visuallyhidden'>page</span>`

I'm not sure there's value of hiding this, I think maintaining the mismatch I think is unhelpful for sighted + AT users.

Also, I noticed this blink-and-you-miss-it extra hover style text-underline next to the arrow. That shouldn't be a blocker but would be good to remove as well if possible.

![Mar-15-2021 18-54-43](https://user-images.githubusercontent.com/71266765/111206866-dd6c2800-85c0-11eb-92c4-e5ea30b421d5.gif)